migration/migrate.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+"""
+Script to store clinical trial print pages in an S3 bucket.
+
+The datafile (supplied as the only command-line parameter) is made up of
+multiple records, one record per line.  Fields in the individual records
+which may contain commas are surrounded by double-quotes, with any double-
+quotes in the field's value replace by a pair of double-quotes.
+(This is really just the output of the MS SQL Server Managent Studio's
+"Save as CSV" on a query.)
+
+Fields in the record are:
+
+key - Unique string (a GUID) for identifying the record.
+cacheDate - Timestamp identifying when the record was originally generated.
+trialIDs - Comma-separated list of trial IDs.
+searchParams - A string containing the JSON search criteria.
+content - A blob of HTML representing the actual page.
+"""
+
+import csv
+import os
+import sys
+
+import boto3
+from botocore.exceptions import ClientError
+
+S3_CLIENT = boto3.client("s3")
+
+if len(sys.argv) != 2:
+    raise RuntimeError("Please provide the input file name as a command line argument.")
+
+
+if (
```

Comment:
I think you can condense to:

`not os.environ.get("CTS_BUCKET_NAME", "").strip()`

With the caveat that a `None` os.environ object (I don't think that's possible?) will throw an error.